src/connectors.rs

```diff
@@ -353,7 +354,9 @@ impl Context for ConnectorContext {
 /// Connector instance status report
 #[derive(Debug, Serialize)]
 pub struct StatusReport {
-    /// connector instance url
+    /// connector alias without the containing flow alias
+    /// when we report this report will happen in the context of a flow,
+    /// so we don't need to provide that extra context here again
     pub(crate) alias: String,
```

Comment:
For consistency would it make sense to still carry over the Alias here instead of a `String`?

Reply:
This struct is directly serialized as part of our API response. The connectors are always reported in the context of a flow, so i think their aliases should be listed without the flow alias here, hence only a string. I could add the `ConnectorId` here but then would have to introduce a new struct for the API which serializes to the expected format.

Reply:
Oh I understand the problem, thanks for the explenation. Would [`#[serde(serialize_with = "path")]`](https://serde.rs/field-attrs.html#serialize_with) perhaps work? We could keep it a `ConnectorId` and won't need to introduce a special struct, just a function that serializes it 


```rust
serialize_no_flow<S>(x: &ConnectorId, s: S) -> Result<S::Ok, S::Error>
where
    S: Serializer,
{
    s.serialize_str(x.connector_alias)
}
```

Reply:
Then i'd rather provider another struct just for the API.